src/inputs/input_field.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export function InputField({ placeholder, value, handleInputTwo }) {
```

Comment:
O nome da prop ```handleInputTwo``` deve ser genérico ou auto-explicativo de um modo que a pessoa entenda o que o componente quer dizer. Se o componente X é usado apenas por um componente Y, faz sentido colocar o nome da própria função como prop da outra. Exemplo: ```handleInput```. Contudo, se o componente X for ser usado por X, Y, Z, vale a pena colocar um nome de prop/função que ressalte o que ele faz, como por exemplo: ```onClickMethod``` ou ```onClickCallback```.